### ISSUE TITLE
display of images or videos on questions page

### ISSUE THREAD

**Issue by davidmooreppf** [opengovernment/askthem-legacy#245]
If a user includes an image in her question, will it appear on q detail page below "full q" body? 

Doesn't nec. need to appear on q listing pages for launch. @walter I know you were working on this last week.

**Comment by walter** [comment]
@davidmooreppf if there is an image or video, uploaded or through a submitted URL, it will show show up.

However, it could use styling by @acacheung.

**Comment by walter** [comment]
One caveat is that they are through a CDN, so in development mode they show up as missing images.

@acacheung for your work, just drop the cdn_ from the image_tag or video_tag temporarily and then return it before you commit and push your changes.

**Comment by davidmooreppf** [comment]
Back to Walter - does this need a deploy? Perhaps. Tested just now on preview and got error msg. below :: 

Media Failed to thumbnail with ffmpegthumbnailer. Check ffmpegthumbnailer install and verify video is not corrupt. Original error: No such file or directory - ffmpegthumbnailer -i /u/apps/askthem/releases/20140109234925/tmp/uploads/1389645475-7877-8112/thumb_Wall_of_Brauny.png -o /u/apps/askthem/releases/20140109234925/tmp/uploads/1389645475-7877-8112/tmpfile.png -c png -q 10 -s 192 -f

**Comment by walter** [comment]
Yeah, I've received that too. There is something different in production that is inhibiting the creation of the thumbnail for the video. I'll have to debug it.

Will keep you posted.

**Comment by davidmooreppf** [comment]
Just noting this still needs a debug. If we can't figure it out, we can comment it out for now on Q form. But keeping it in is great if we can fix it. 

Media Failed to thumbnail with ffmpegthumbnailer. Check ffmpegthumbnailer install and verify video is not corrupt. Original error: No such file or directory - ffmpegthumbnailer -i /u/apps/askthem/releases/20140122213339/tmp/uploads/1390429681-22163-4387/thumb_Poker_Dogz.png -o /u/apps/askthem/releases/20140122213339/tmp/uploads/1390429681-22163-4387/tmpfile.png -c png -q 10 -s 192 -f

**Comment by walter** [comment]
On my list.

It actually is pretty high priority as it may involve @rchekaluk doing some behind-the-scenes tweaking of the ffmpegthumbnailer. Will take a look tonight or tomorrow.

**Comment by walter** [comment]
This is in place and working on preview. It's deployed to production.

@davidmooreppf all that remains is testing with a real question on production. I would hold off on that until the staff member "delete question" functionality is in place.

**Comment by walter** [comment]
@davidmooreppf, delete question is implemented in #266. Will need to repeat registering of staff members on production, then let me know, I'll grant staff member status, then you can delete questions on both preview and production.

Waiting on some ops stuff for @rchekaluk (sidekiq restart not working).

**Comment by walter** [comment]
@davidmooreppf & @maryamgunja you can test this now on the live site.

Once done, delete the question.

**Comment by walter** [comment]
@davidmooreppf ^

**Comment by walter** [comment]
See #333 about issue with showing Vimeo videos. @davidmooreppf please use this issue for a discussion of that problem and further QA of this feature.
